### ISSUE TITLE
Link to New Production Site

### ISSUE THREAD

**Issue by scooterbee** [amplify-technologies/lawns-website#6]
Before we link to the new production site, be sure that the following are completed:

- [ ] SMS has to work
- [ ] Email/Sendgrid has to work

**Comment by navanjr** [comment]
@amplify-technologies/report What about existing data located in the `prod-lawns` environment? Will it need to be copied to `amplify-lawns`

**Comment by scooterbee** [comment]
Yes.  We will at least want to migrate some, if not all, of the existing
data into amplify-lawns.
